devel/asio/Portfile

```diff
@@ -2,26 +2,30 @@
 
 PortSystem                  1.0
 PortGroup                   openssl 1.0
+PortGroup                   muniversal 1.0
 
 name                        asio
-version                     1.20.0
+version                     1.22.1
 revision                    0
-checksums                   rmd160  762d0c9dec18b7ce0fdf9069067900ab1345887f \
-                            sha256  204374d3cadff1b57a63f4c343cbadcee28374c072dc04b549d772dbba9f650c \
-                            size    1845826
+checksums           rmd160  02ed50e944d1ebe835fa15d0e90dfd46c974ecf5 \
+                    sha256  6874d81a863d800ee53456b1cafcdd1abf38bbbf54ecf295056b053c0d7115ce \
+                    size    3085878
 
 license                     Boost-1
 description                 Asio C++ Library.
 long_description            Asio is a cross-platform C++ library for network and low-level I/O \
-                            programming that provides developers with a \
-                            consistent asynchronous model using a modern C++ approach.
+                            programming that provides developers \
+                            with a consistent asynchronous model using a modern C++ approach.
 maintainers                 {gmail.com:g.litenstein @Lord-Kamina} openmaintainer
 categories                  devel
 
 homepage                    https://think-async.com/Asio/
 master_sites                sourceforge:project/${name}/${name}/${version}%20%28Stable%29
 use_bzip2                   yes
 
+# Tarball is missing `src/examples/cpp20`, which causes error
```

Comment:
did you report this to upstream?

Reply:
I’d be surprised if that has not been done already (I am not the first to discover the problem), but I can check it.

Reply:
@reneeotten Let me actually move to Github from Sourceforce. The former has a tarball without examples missing: https://github.com/chriskohlhoff/asio/releases/tag/asio-1-22-1